servicepulse/intro-pending-retries.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Pending Retries Message Management in ServicePulse
+summary: Describes how ServicePulse monitors archived failed messages, and allows unarchiving archived failed messages.
+tags:
+ - ServicePulse
+reviewed: 2016-07-28
+---
+
+
+In Versions **1.6.6** and above ServicePulse includes a screen to view and manage failed messages that have been sent for retry but are pending final status.
+
+Pending retries messages can be found by going to the pending retries screen.
+
+![Pending Retries Tab](images/pending-retries.png 'width=500')
+
+
+### Pending Retries Messages
+
+The Pending Retries Messages screen shows failed messages which have been retried but the status of that retry is pending. The status of retried failures is updated when either the message is processed again as either an Audited message (success) or as a failed message.
+
+Failed messages that are retried may stay in the pending state for the following reasons:
+ - The retrying endpoint is not working (e.g. crashed or is scaled-out) and the retried messages is waiting in the queue and has not yet been processed.
+ - The retrying endpoint does not have auditing enabled but has successfully processed the retried message.
```

Comment:
What is the scaled-out scenario? I understand the crashed scenario, i.e. process not running. What about the scaled out scenario. In this case, there are 2 instances running, won't one of them process it? Can you clarify?

I am not sure if calling the endpoint as the retrying endpoint is right. How about:

Failed messages that are retried may stay in the pending state for the following reasons:

- If the endpoint where the retried message is sent to is not running, e.g. crashed, then the retried messages would be waiting in the queue. 
- If auditing is not setup for the endpoint where the retried message is sent to and the message has been successfully processed by that endpoint.